### PR TITLE
Fix: catch unhandled exception in text extractor

### PR DESCRIPTION
**Pull request by broma0**
I've added a try/catch around the source of an unhandled exception that I discovered while textract'ing a few thousand files. If need be, I will find the exact cause and create a test case. Please let me know.

### CODE REVIEW

File: lib/extractors/text.js [hunk 1]
```diff
@@ -10,8 +10,12 @@ function extractText( filePath, options, cb ) {
       cb( error, null );
       return;
     }
-    encoding = jschardet.detect( data ).encoding.toLowerCase();
-    decoded = iconv.decode( data, encoding );
+    try {
+      encoding = jschardet.detect( data ).encoding.toLowerCase();
+      decoded = iconv.decode( data, encoding );
+    } catch ( e ) {
+      cb( e );
```
Comment (dbashford) [inline lib/extractors/text.js]: This should probably be `return cb( e )`, otherwise the `cb` will be executed twice.

Comment (broma0) [inline lib/extractors/text.js]: Right you are, Harry.

**Comment by dbashford** [comment]
Thanks!

**Comment by dbashford** [comment]
Released as `v2.1.1`

**Pull request merged**